example/main.go

```diff
@@ -3,13 +3,13 @@ package main
 import (
 	"context"
 	"flag"
+	"github.com/devopsfaith/krakend/router"
+	"github.com/khvysofq/krakend-etcd"
```

Comment:
this import points to the fork

Reply:
also, please run the [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) command on the code